docs/getting-started/setup.md

```diff
@@ -93,3 +95,35 @@ create table if not exists `catalog`.`schema`.`dq_stats` (
 );
 ```
 
+### DQ Detailed Stats Table
```

Comment:
Will the data for this stats table also be sent to kafka stream if streaming options are enabled?

Reply:
As of right now, it does not but nothing harm in writing if needed, as it contains only metadata.

Reply:
Yeah, We can take it up as a follow-up issue.